### PR TITLE
Add the mango ingredient (transport by plane)

### PR DESCRIPTION
**Pull request by magopian**
[user story](https://www.notion.so/TRANSPORTS-Ajouter-du-transport-en-avion-par-exemple-en-ajoutant-la-mangue-5d57dee47db04f42b26776a1fbc77922)

### CODE REVIEW

File: public/data/food/ingredients.json [hunk 1]
```diff
@@ -155,5 +155,13 @@
...
+  },
+  {
+    "id": "mango",
+    "name": "Mangue",
+    "default": "c1e6f84a4fa89c2408bd7a2b2ed6cb96",
+    "default_origin": "OutOfEuropeAndMaghrebByPlane",
+    "raw_to_cooked_ratio": 0.856,
+    "variants": {}
   }
 ]
```
Comment (n1k0) [inline public/data/food/ingredients.json:167]: Nit: would it be possible to have the scripts adding a trailing CR? Most editors add one when saving, that would prevent unecessary diffs I think.

File: public/data/food/processes/builder.json [hunk 2]
```diff
@@ -723,6 +723,40 @@
...
+      "pma": 1.5250357563192335e-08,
+      "ior": 0.009312422846439053,
+      "fru": 1.3485429853584745,
+      "mru": 1.2441737422801986e-06,
+      "ldu": 20.785276516354987,
+      "wtu": 0.09768925571584185,
+      "etf": 67.43096463527166,
+      "htc": 7.327931292306994e-11,
+      "htn": 3.0628514383642146e-09,
+      "bvi": 0.0,
```
Comment (n1k0) [inline public/data/food/processes/builder.json:756]: No bvi, is that expected?

**Review by n1k0 — APPROVED** [review]
LGTM. Ideally we should add a unit test and an e2e test to `tests/e2e-food.json` to cover the Mango case.

File: public/data/food/processes/builder.json (hunk 2 above)
Comment (magopian) [reply public/data/food/processes/builder.json:756]: Yeah, we're missing a bunch of bvi data for now, but they'll be added at a later point i'm sure ;)

**Comment by magopian** [comment]
@n1k0 : I just added some tests, and the newlines at the end of the exported json files, would you care giving it a quick look before i land this? 🙏

**Comment by n1k0** [comment]
@magopian I don't see tests in the patch, am I missing something obvious?

**Review by n1k0 — APPROVED** [review]
Looks great, ship it! :rocket:

**Pull request merged**